command-expansion-server/src/getCommandTypes.ts

```diff
@@ -0,0 +1,21 @@
+import type {Pool} from 'pg';
+import fs from 'fs';
+import { ErrorWithStatusCode } from './utils/ErrorWithStatusCode.js';
+
+export async function getCommandTypes(db: Pool, dictionaryId: number): Promise<string> {
+
+  const {rowCount, rows} = await db.query(`
+    SELECT command_types
+    FROM command_dictionary
+    WHERE id = $1;
+  `, [
+    dictionaryId
+  ]);
+
+
+  const [row] = rows;
+  if (rowCount < 0) {
+    throw new ErrorWithStatusCode(`No dictionary with id: ${dictionaryId}`, 404);
```

Comment:
Is it possible for `rowCount` to be `< 0`? If I'm understanding its assignment correctly it's a count on the `SELECT` query, right? If so, then `< 0` should not be possible, only `== 0` would be. Good chance I'm just missing something crucial here.

Reply:
In `getExpansionSet.ts` `rows.length < 1` is used, which is what I would've expected here I think.

Reply:
100% correct

Reply:
Feel free to update. I think I've tried to deprecate out the rowcount pattern in place of rows.length as well

Reply:
I think this was the result of the rebase with Ryan's branch

Reply:
Cool, I don't see an update pushed yet but feel free to resolve this convo. when you've changed the condition. :)

Reply:
Any chance you have time to make that update? I need to focus on M2020 today

Reply:
Ohh I misunderstood, yeah I'll push this.

Reply:
Just pushed bcc4ea311.